index.js

```diff
@@ -67,6 +71,54 @@ function toggleColor(element) {
     }
 }
 
+header.addEventListener('click', function(){
+    if (header.style.color === 'black'){
+        header.style.color = 'red'
+    } else {
+        header.style.color = 'black'
+    }
+    
+})
+// didnt see toggleColor() when I originally wrote the function uptop. I'd basically take
+// that and send it into the listener method...please verify if the following would be correct.
+// addEventListner ('click',toggleColor)
```

Comment:
Good catch! yup, but don't forget that `toggleColor` is expecting an `element` as an argument. What would make sense to pass to it when we call it?